Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt

```
@@ -44,7 +44,7 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
     )
     ly_add_googletest(
         NAME Gem::AtomViewportDisplayInfo.Tests
-        
+        LABESL REQUIRES_tiaf
```

There is a typo in `LABEL` here
```suggestion
        LABEL REQUIRES_tiaf
```